api.yaml

```diff
@@ -678,6 +678,121 @@ paths:
             $ref: '#/components/responses/ExecutionDoesNotExist'
           '429':
             $ref: '#/components/responses/TooManyRequests'
+    /v2/runs/executions/{executionId}:
+      get:
+        summary: Get execution results
+        description: Get the result of an execution, together with its tests, for failed tests a list of the failed steps
```

Comment:
I think it was decided to only return one list which has all test results, of all statuses.

Reply:
that is what i return but Ran said that for failed tests we should return the failed steps. I will check with him again.

Reply:
spoke with Ran he asked to that we keep the same format for failed tests.
@liorama